code/modules/reagents/reagent_containers/glass.dm

```diff
@@ -168,8 +168,50 @@
 	materials = list(MAT_GLASS=2500)
 	volume = 100
 	amount_per_transfer_from_this = 10
-	possible_transfer_amounts = list(5,10,15,20,25,30,50,100)
+	possible_transfer_amounts = list(5,10,15,20,25,30,50,75,100)
 	flags_1 = OPENCONTAINER_1
+	w_class = WEIGHT_CLASS_SMALL
+
+/obj/item/reagent_containers/glass/beaker/vial
+	name = "vial"
+	desc = "A vial. Can hold up to 25 units."
+	icon_state = "minivial"
+	materials = list(MAT_GLASS=250)
+	volume = 25
+	amount_per_transfer_from_this = 10
+	possible_transfer_amounts = list(5,10,15,20,25)
+	flags_1 = OPENCONTAINER_1
+
+/obj/item/reagent_containers/glass/beaker/erlenmeyer
+	name = "erlenmeyer flask"
+	desc = "An erlenmeyer flask. Can hold up to 75 units."
+	icon_state = "eflask"
+	materials = list(MAT_GLASS=1000)
+	volume = 75
+	amount_per_transfer_from_this = 10
+	possible_transfer_amounts = list(5,10,15,20,25,30,50,75)
+	flags_1 = OPENCONTAINER_1
+
+/obj/item/reagent_containers/glass/beaker/round
+	name = "round-bottomed flask"

```

Comment:
feels like a round bottom shouldn't be able to be placed on a surface without automatically tipping.

Reply:
I'unno, I want more visual beaker variety to spice up chem. Don't want to make people never use a kind of beaker because it sucks.

Reply:
It could be used interestingly, in that its got a higher capacity, but you have to hold onto it, placing it spills it.

Reply:
Give the huge round beaker a boost, but if you put it down or drop it it spills?